image-loader/app/model/UploadStatus.scala

```diff
@@ -0,0 +1,20 @@
+package model
+
+import play.api.libs.json._
+
+case class UploadStatus(
+                         id: String,
+                         uploadedBy: String,
+                         fileName: String,
+                         status: String
+                       )
```

Comment:
We will need to include an error field on the UploadStatus so as to tell users what exactly it was that went wrong.